src/tox/config/__init__.py

```diff
@@ -1924,8 +1944,11 @@ def processcommand(cls, reader, command, replace=True):
                     continue
 
                 new_arg = ""
-                new_word = reader._replace(word)
-                new_word = reader._replace(new_word)
+                has_dual_quote = "''" in word
+                new_word = reader._replace(word, unquote_path=False)
+                new_word = reader._replace(new_word, unquote_path=False)
+                if not has_dual_quote:
+                    new_word = new_word.replace("''", "'")
```

Comment:
this has the potential for changing the behaviour of any existing `tox.ini` which used quotes and also had quotes inserted via substitution.  The most obvious case would be an env var of `'` and commands using the envvar and quotes.  Then, does stripping dual quotes change the behaviour of those previous usages?  In theory, this removal would be simply merging quoted strings.  Need to think more about possible breakages.

Reply:
Ah, this simple replace would break `\\''`.  Easy fixed with a `re`.

Reply:
`\\''` now fixed.  No test coverage yet.  Other obscure oddities may still exist.